### PR TITLE
Remove legacy devhub

### PR DESCRIPTION
**Pull request by EnTeQuAk**
Our new devhub pages are live for quite a long time now, let's remove the waffle switch and old code.

This also tries to remove some unused css and removes some images that aren't in use anymore.

### CODE REVIEW

**Comment by eviljeff** [comment]
Issue plz.

**Comment by EnTeQuAk** [comment]
> Issue plz.

https://github.com/mozilla/addons-server/issues/5582

**Comment by EnTeQuAk** [comment]
> Looks fine. What's still in progress?

Nothing anymore, I just forgot that I put that into the title. Ready to review!

File: src/olympia/devhub/templates/devhub/index.html [hunk 1]
```diff
@@ -1,5 +1,81 @@
...
+      {% include 'devhub/new-landing/components/my-addons.html' %}
+    {% endif %}
+
+    {% include 'devhub/new-landing/components/footer.html' %}
+    <div id="tooltip">
+        <span></span>
+    </div>
+    {% block site_js %}
+      <script src="{{ CDN_HOST }}{{ cache_buster(url('jsi18n')) }}"></script>
+      {{ js('common') }}
```
Comment (eviljeff) [inline src/olympia/devhub/templates/devhub/index.html]: Out of scope for this patch, but there's a whole mountain of javascript in 'common' - starting a new minimal bundle would be better as it doesn't use any of the existing templates so require anything in those.

Comment (EnTeQuAk) [inline src/olympia/devhub/templates/devhub/index.html]: yeah, I thought about it but then, `common` is usually already cached on the users machines so I think the gain would be minimal, especially when the user downloads the whole `common` bundle once he clicks on "View all submissions" or other links on the page.

**Pull request merged**